examples/vmi-sriov.yaml

```diff
@@ -30,16 +30,16 @@ spec:
   - name: default
     pod: {}
   - multus:
-      networkName: sriov/sriov-network
+      networkName: sriov-network
     name: sriov-net
   terminationGracePeriodSeconds: 0
   volumes:
   - containerDisk:
-      image: registry:5000/kubevirt/fedora-cloud-container-disk-demo:devel
+      image: registry:5000/kubevirt/fedora-cloud-sriov-lane-container-disk:devel
```

Comment:
this one doesnt work, use please
`registry:5000/kubevirt/fedora-sriov-lane-container-disk:devel`

Reply:
can you create a PR to fix this file please?
we shouldnt block on it

Reply:
Done

Reply:
`image: registry:5000/kubevirt/fedora-cloud-sriov-lane-container-disk:devel` is NA
Try to use this and see the image is NA
the right image is 
https://github.com/kubevirt/kubevirt/pull/4356#discussion_r508334326
so i guess you can change the code that creates this file ?

Reply:
Oh now I get it, no problem I will change it